### DISCUSSION TITLE
not able to download

### DISCUSSION THREAD

**Discussion by 365krypto** [lllyasviel/Fooocus#3796] [Q&A] [answered]
hello all I am having a hard time with the final step. In the Command it is telling me I don't have Nvidia driver and Ive tried multiple drivers and none of them let me download but my computer isn't the best but its no slouch either. Ive included what its telling me and my computer specs. What am I missing please help. 
![Screenshot 2024-12-17 224847](https://github.com/user-attachments/assets/c5a25802-6860-4f59-85d8-d334a2b0a280)
![Screenshot 2024-12-17 224907](https://github.com/user-attachments/assets/0d6dd7e7-df84-46a7-b91c-9d376dc4a925)

**Comment by Yogesh-DevHub** [comment]
After the driver, that program is also asking to 'Check if you have a Nvidia GPU'

**Comment by JT-Gresham** [comment, marked as answer]
Just from what I can see...and making a couple assumptions here....I see that you are on a laptop...I see you have a Ryzen APU which explicitly states you have Radeon graphics...

Since most laptops use integrated graphics, in this case, your Radeon integrated graphics, I'd guess you don't have an external GPU...let alone an Nvidia one...

So...in your installation, you didn't specify AMD rather than Nvidia?  For AMD and Intel setups are different (and more problematic) since most of the AI software out there likes to utilize CUDA...which is proprietary Nvidia software.

You'll need to look up how to get this working with your AMD setup, most likely....just like I did with my Intel Arc setup...

Good luck...hope this info helped point you in the right direction...
